docs/releases/4.0.md

```diff
@@ -162,3 +162,7 @@ If your code contains references to URL route names within the `wagtaildocs` or
 * `wagtaildocs:chooser_upload` is now `wagtaildocs_chooser:create`
 * `wagtailsnippets:list`, `wagtailsnippets:list_results`, `wagtailsnippets:add`, `wagtailsnippets:edit`, `wagtailsnippets:delete-multiple`, `wagtailsnippets:delete`, `wagtailsnippets:usage`, `wagtailsnippets:history`: These now exist in a separate `wagtailsnippets_{app_label}_{model_name}` namespace for each snippet model, and no longer take `app_label` and `model_name` as arguments.
 * `wagtailsnippets:choose`, `wagtailsnippets:choose_results`, `wagtailsnippets:chosen`: These now exist in a separate `wagtailsnippetchoosers_{app_label}_{model_name}` namespace for each snippet model, and no longer take `app_label` and `model_name` as arguments.
+
```

Comment:
Needs a note about the URL name changes (`wagtailimages:chooser` -> `wagtailimages_chooser:choose` etc)

Reply:
Good catch! Added in d3ebd24741aeb4c258af785ef7688f43ae3c5af8.